packages/mobile/src/components/user/FollowButton.tsx

```diff
@@ -59,3 +60,48 @@ export const FollowButton = (props: FollowButtonsProps) => {
     />
   )
 }
+
+type FollowButton2Props = Partial<ButtonProps> & {
+  userId: ID
+  noIcon?: boolean
+  style?: StyleProp<ViewStyle>
+  followSource: FollowSource
+}
+
+export const FollowButton2 = (props: FollowButton2Props) => {
```

Comment:
do we want to keep the 2 name?

Reply:
oh actually its not used in many places, ill migrate everything